README.md

```diff
@@ -24,5 +24,5 @@ List of college projects by undergraduates from 076 Batch, Pulchowk Campus.
 Follow this format while adding your project:
 
 ```
-* [name of project](link to the project) - @username (roll number)
+* [name of project](link to the project) - @username (roll number), @username (roll number) *(add according to the number of students)*
```

Comment:
*(add according to the number of students who worked on the project, multiple entries allowed)*

Reply:
yes, think we should add this, a few people were asking me (sometime back) if the repo was in person X's repository then whether or not they could write person Y's name too, this clears the confusion

Reply:
yes, this is good